security-questionnaire.md

```diff
@@ -2,35 +2,76 @@
 
 01.  What information might this feature expose to Web sites or other parties,
      and for what purposes is that exposure necessary?
+
+    - This feature allows web developers to establish WebTransport over HTTP/3
+      sessions to servers. For the purpose this feature exposes the network
+      circumstances and error information to some extent.
 02.  Do features in your specification expose the minimum amount of information
       necessary to enable their intended uses?
+
+    - Yes. Also, we're careful of not exposing information for endpoints that
```

Comment:
I suggest linking to https://w3c.github.io/webtransport/#protocol-security here.

Reply:
Done.